homeworks/gaalistvantamas/4_exceptions_logging/to_do.py

```diff
@@ -0,0 +1,98 @@
+"""
+Author: Gaál István Tamás
+Task: Homework-4
+"""
+import logging
+
+# log setup
+LOG_FILE_PATH = "homeworks/gaalistvantamas/4_exceptions_logging/todo.log"
+
+file_handler = logging.FileHandler(LOG_FILE_PATH)
+stream_handler = logging.StreamHandler()
+
+formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+
+file_handler.setFormatter(formatter)
+stream_handler.setFormatter(formatter)
+
+file_handler.setLevel(logging.INFO)
+stream_handler.setLevel(logging.INFO)
+
+logger = logging.getLogger()
+logger.addHandler(file_handler)
+logger.addHandler(stream_handler)
+logger.setLevel(logging.INFO)
+
+# Task Manager
+
+FILE_PATH = "homeworks/gaalistvantamas/4_exceptions_logging/todo.txt"
+
+try:
+    with open(FILE_PATH, "x") as file:
+        file.write("")
+except:
+    print("The file is already exist.")
+
+def display_menu():
+    print("\n1. Add Task")
+    print("2. View Task")
+    print("3. Remove Task")
+    print("4. Exit\n")
+
+def add_task(task: str):
+    try:
+        with open(FILE_PATH, "a") as file:
+            file.write(f"{task}" + ",")
+    except:
+        print("Add task Fail!")
+
+    logger.info(f"Task {task} is added!")
+
+def view_task():
+    try:
+        with open(FILE_PATH, "r") as file:
+            tasks = file.read()
+            task_list = list(tasks.split(","))
+            for task in task_list:
+                print(f"{task.strip()}")
+    except:
+        print("File is not exist or reading problems!")
+
+def remove_task(removable_task: str):
+    try:    
+        with open(FILE_PATH, "r") as file:
+            tasks = file.read()
+            task_list = list(tasks.split(","))
+
+            for task in task_list:
+                print(f"{task.strip()}")
+                if task == removable_task:
+                    task_list.remove(task)
+            
+            with open(FILE_PATH, "w") as file:
+                for task in task_list:
+                    file.write(f"{task}" + ",")
+    except:
+        print("Remove task Fail!")
+    
+    logger.info(f"Task {removable_task} is removed!")
```

Comment:
Ez itt akkor is kiírja hogy task is removed, ha amit megadtam nincs is a listában. Ezt a visszajelzést inkább a 71. soron írd ki, ahol ténylegesen megtörténik a törlés.